### ISSUE TITLE
[BUG] [Linter] false positive on H037 for nested attributes

### ISSUE THREAD

**Issue by christopherpickering** [Riverside-Healthcare/djLint#642]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

```html
<link rel="stylesheet" href="styles.css" media="print" onload="this.media='all'"/>
```
triggers h037 with the nested this.media.
